frontend/src/lib/schemas/response/LearningKitRes.ts

```diff
@@ -1,9 +1,11 @@
 import { z } from 'zod';
+import {LearningUnitResSchema} from "$lib/schemas/response/LearningUnitRes";
 
 export const LearningKitResSchema = z.object({
 	uuid: z.string().uuid(),
 	name: z.string().nonempty(),
-	description: z.string().nullable(),
+	learningUnits: z.array(LearningUnitResSchema).optional(),
+	description: z.string().optional().nullable(),
```

Comment:
I think the BE sends null values, not optioonal values to optional seems redundant in Response schemas, and nullable should be used instead